SQL/db/migrations/02_add_time_to_peep.sql

```diff
@@ -1 +1 @@
-ALTER TABLE chitter ADD COLUMN time VARCHAR(100);
\ No newline at end of file
+ALTER TABLE chitter ADD COLUMN created_at VARCHAR(100);
```

Comment:
Great that you've added these setup instructions to the migrations folder so that another user can follow the same database setup and any modifications to the tables.